src/program/snabbnfv/doc/getting-started.md

```diff
@@ -388,9 +388,9 @@ $ sudo /usr/local/bin/qemu-system-x86_64 \
   -drive if=virtio,file=/home/mwiget/ubuntu2.qcow2 -M pc -smp 1 \
   --enable-kvm -cpu host -m 1024 -numa node,memdev=mem \
   -object memory-backend-file,id=mem,size=1024M,mem-path=/mnt/huge,share=on \
-  -chardev socket,id=char0,path=/home/mwiget/vhost-sockets/vm2.socket,server \
-  -netdev type=vhost-user,id=net0,chardev=char0 \
-  -device virtio-net-pci,netdev=net0,mac=52:54:00:00:00:02 \
+  -chardev socket,id=char1,path=/home/mwiget/vhost-sockets/vm2.socket,server \
+  -netdev type=vhost-user,id=net1,chardev=char1 \
+  -device virtio-net-pci,netdev=net1,mac=52:54:00:00:00:02 \
```

Comment:
But these IDs are passed to 2 different QEMU instances which have their own numbering and are independent. So it shoudl be perfectly fine to use them as they are now. Or am I wrong?